addons/medical_gui/initSettings.sqf

```diff
@@ -126,3 +126,12 @@ private _categoryColors = [ELSTRING(medical,Category), format ["| %1 |", LELSTRI
     true,
     true // isGlobal
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(peekMedicalInfoReleaseDelay),
+    "SLIDER",
+    [LSTRING(PeekMedicalInfoReleaseDelay_DisplayName), LSTRING(PeekMedicalInfoReleaseDelay_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    [0, 5, 1, 0],
```

Comment:
```suggestion
    [0, 5, 1, -1],
```

Reply:
Oh nice! I though there had to be a way to do an integer slider.